core/src/main/java/smile/base/cart/CART.java

```diff
@@ -271,12 +271,12 @@ protected boolean split(final Split split, PriorityQueue<Split> queue) {
         int mid = split.lo + trueSamples.length;
 
         LeafNode trueChild = newNode(trueSamples);
-        assert trueChild.size == split.trueCount : String.format("trueChild.size != split.trueCount: %d != %d", trueChild.size, split.trueCount);
-        assert trueChild.size >= nodeSize : String.format("trueChild size is too small: %d < %d", trueChild.size, nodeSize);
+        //assert trueChild.size == split.trueCount : String.format("trueChild.size != split.trueCount: %d != %d", trueChild.size, split.trueCount);
```

Comment:
this is not intended change, but pending on resolution for https://github.com/haifengl/smile/issues/526